denizen_scripts/survival/repo-link/shops/specialty_shop.dsc

```diff
@@ -0,0 +1,68 @@
+specialty_shop_assignment:
+    type: assignment
+    actions:
+        on assignment:
+        - trigger name:click state:true
+        on click:
+        - inject specialtyInventoryInject
+
+specialtyInventoryInject:
+    type: task
+    script:
+    - define inventory <inventory[specialtyInventory]>
+    - give "<item[player_head].with[skull_skin=<player.skull_skin>;display_name=<&e><&o><player.display_name>;lore=<&a>Money: <&e><player.money>]>" to:<[inventory]> slot:5
+    - inventory open d:<[inventory]>
```

Comment:
If this is the only time this script is used, it's a better idea to just consolidate the script within the single script container.
```suggestion
        - define inventory <inventory[specialtyInventory]>
        - give "<item[player_head].with[skull_skin=<player.skull_skin>;display_name=<&e><&o><player.display_name>;lore=<&a>Money: <&e><player.money>]>" to:<[inventory]> slot:5
        - inventory open d:<[inventory]>
```